securedrop_client/gui/widgets.py

```diff
@@ -790,35 +795,35 @@ def __init__(self, source: Source):
         summary_layout.setSpacing(0)
         self.name = QLabel()
         self.name.setObjectName('source_name')
-        self.preview = QLabel('')
+        self.preview = QLabel()
         self.preview.setObjectName('preview')
+        self.preview.setFixedSize(QSize(365, 60))
         self.preview.setWordWrap(True)
         summary_layout.addWidget(self.name)
         summary_layout.addWidget(self.preview)
 
         # Set up metadata
         self.metadata = QWidget()
         self.metadata.setObjectName('metadata')
+        self.metadata.setMaximumWidth(30)
         metadata_layout = QVBoxLayout(self.metadata)
         metadata_layout.setContentsMargins(0, 0, 0, 0)
         metadata_layout.setSpacing(0)
-        self.attached = SvgLabel('paperclip.svg', QSize(16, 16))
+        self.attached = SvgLabel('paperclip.svg', QSize(14, 16))
         self.attached.setObjectName('paperclip')
-        self.attached.setFixedSize(QSize(20, 20))
-        spacer = QWidget()
-        metadata_layout.addWidget(self.attached, 1, Qt.AlignRight)
-        metadata_layout.addWidget(spacer, 1)
+        metadata_layout.addWidget(self.attached)
+        metadata_layout.addStretch()
```

Comment:
seems like we need a bit of layout refactoring, i.e. should we put `updated` timestamp in this layout? reason being is that in the zeplin spec the timestamps are inline with the preview text and roughly centered in the source list item - well, 31px above, 38px below - and directly under the paperclip (we could put in a followup but since we're working on it now and it would not be a huge change we could resolve as part of this PR - let me know what you think)

Reply:
Since the dates are not a fixed size and I think we also agreed to include a two-digit year, which is not shown in Zeplin yet, we're not quite ready to have our timestamps show up in a fixed size column. I think there might be an issue for this open somewhere (@eloquence would know!), but if now I'll add it to https://github.com/freedomofpress/securedrop-client/issues/135

Reply:
The _date_-stamps were designed specifically to the format shown in Zeplin (DD Mmm)—which would require use (or creation) of a different library. Allie and I discussed this some time ago, and I forget what the exact resolution was... but I'm assuming we'd agreed to punt on perfecting their display in the _Source list_, until it could be shown in the `DD Mmm` date-stamp fashion. 

Allie said she knew of a library that shows some months as 4-letter abbreviations and others as 3-letter abbreviations, but I want to keep all of them restricted to 3-letters for the month, and 2-letters for the date. That uniformity is important to facilitate compact _Source_ item footprints that are also visually clean and easy to scan. 

Otherwise I think I would have re-done all of the mockups to show the datestamp as a timestamp, and in a footprint able to extend to show the "2 minutes ago" length ones.

Reply:
The recap of our last discussion on how to present dates in different parts of the UI is here:
https://github.com/freedomofpress/securedrop-client/issues/332#issuecomment-487667668